.github/workflows/ci.yml

```diff
@@ -1,8 +1,5 @@
 name: CI
 
-env:
-  VOLTA_FEATURE_PNPM: 1
```

Comment:
I don't think that was used (anymore)!?

Reply:
correct, this was hanging around from before 👍